Packages/io.chainsafe.web3-unity.web3auth/Runtime/chainsafe.web3-unity.web3auth.asmdef

```diff
@@ -2,7 +2,9 @@
     "name": "chainsafe.web3-unity.web3auth",
     "rootNamespace": "",
     "references": [
-        "GUID:5426c6b788696eb4c88f4198b59839eb"
+        "GUID:5426c6b788696eb4c88f4198b59839eb",
+        "GUID:0c370a41a98240ff8e4dd4b17c2813bf",
```

Comment:
This is a reference to the samples assembly definition, it should be removed

Reply:
i need it for the sample output util but i can just change them to debugs, ez

Reply:
You can't reference Samples in Package because this will throw a compile error unless samples are imported, which isn't always the case

Reply:
yeah, i was being lazy there for sure, fixed